arcade/sprite.py

```diff
@@ -1084,14 +1084,15 @@ def draw(self, *, filter=None, pixelated=None, blend_function=None) -> None:
 
     def draw_hit_box(self, color: Color = BLACK, line_thickness: float = 1) -> None:
         """
-        Draw a sprite's hit-box.
+        Draw a sprite's hit box.
 
-        The 'hit box' drawing is cached, so if you change the color/line thickness
-        later, it won't take.
+        The hit box drawing is cached, so if you change the color or line thickness
+        later, it won't take more time.
```

Comment:
The original phrase was to let the user know, if they later changed the color from red to blue, the caching would prevent the update from happening at all. Not that it wouldn't take more time. And from einarf's comment, I don't think this is the case so this can just be removed.